tests/integration/test_charm.py

```diff
@@ -144,7 +176,7 @@ async def test_with_ingress(
     # mypy doesn't see that ActiveStatus has a name
     await model.wait_for_idle(status=ActiveStatus.name)  # type: ignore
 
-    traefik_ip = next(await get_unit_ips(traefik_app_name))
+    traefik_ip = (await get_unit_ips(traefik_app_name))[0]
```

Comment:
Any reason for this change ?

Reply:
The new one is just simpler.

Reply:
Ok, that probably just me who's not fan of accessing arrays by index :)